app/presenters/users/name_presenter.rb

```diff
@@ -21,7 +21,13 @@ def last_first_name
     def render(name)
       parts = Array(name)
       parts << "(#{username})" if username_label?
-      parts << "(#{user.class.human_attribute_name(:suspended)})" if suspended_label? && suspended?
+      if suspended_label?
+        if suspended?
```

Comment:
I just realized that this may not be the behavior we want if the user was suspended by an admin and then later gets removed from LDAP. This would have them showing the `SUSPENDED` label. I've asked the client for a preference of precedence.